Changelog.md

```diff
@@ -1,8 +1,8 @@
 [0.11.0.0] — September 2020
  * [Change internal representation of `ByteString`, removing offset](https://github.com/haskell/bytestring/pull/175)
-   * The old `PS` constructor has been turned into a pattern synonym that is available with GHC >= 8.0 for backwards compatibility.
+   * The old `PS` constructor has been turned into a pattern synonym that is available with GHC >= 8.0 for backwards compatibility. Consider adding `if !impl(ghc >=8.0) { build-depends: bytestring < 0.11 }` to packages, which use `PS` and still support GHC < 8.0.
```

Comment:
```suggestion
   * The old `PS` constructor has been turned into a pattern synonym that is available with GHC >= 8.0 for backwards compatibility. Consider adding `if impl(ghc < 8.0) { build-depends: bytestring < 0.11 }` to packages, which use `PS` and still support GHC < 8.0.
```
If `<` is also supported (IIRC it is), I think it may be better to avoid the negation...

Reply:
No. The `!impl(ghc >= 8.0)` and `impl(ghc < 8.0)` are different. Latter is true only when implementation is GHC, so the former is more correct:

If implementation is not GHC, and particularly not GHC-8.0 or later.